examples/app-router/app/my-user-profile/page.tsx

```diff
@@ -1,5 +1,7 @@
+'use client'
```

Comment:
Why do we need use client directive here? 

The widget is already a client component